File: app/src/main/java/com/example/vkwall/data/model/GetWallByID/ResponseById.java
package com.example.vkwall.data.model.GetWallByID;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class ResponseById {

    @SerializedName("items")
    private List<ItemById> items;
    @SerializedName("profiles")
    private List<ProfileById> profiles;
    @SerializedName("groups")
    private List<GroupById> groups;

    public List<ItemById> getItems() {
        return items;
    }

    public void setItems(List<ItemById> items) {
        this.items = items;
    }

    public List<ProfileById> getProfiles() {
        return profiles;
    }

    public void setProfiles(List<ProfileById> profiles) {
        this.profiles = profiles;
    }

    public List<GroupById> getGroups() {
        return groups;
    }

    public void setGroups(List<GroupById> groups) {
        this.groups = groups;
    }

}

File: app/src/main/java/com/example/vkwall/data/model/GetWallByID/ResponseById.kt
package com.example.vkwall.data.model.GetWallByID

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class ResponseById {

    @SerializedName("items")
    var items: List<ItemById>? = null
    @SerializedName("profiles")
    var profiles: List<ProfileById>? = null
    @SerializedName("groups")
    var groups: List<GroupById>? = null

}
